tests/common/dualtor/dual_tor_io.py

```diff
@@ -365,6 +393,161 @@ def generate_from_server_to_t1(self):
         self.sent_pkt_dst_mac = self.vlan_mac
         self.received_pkt_src_mac = [self.active_mac, self.standby_mac]
 
+    def generate_from_t1_to_soc(self):
```

Comment:
There are code duplication between those four methods:
* `generate_from_t1_to_soc`
* `generate_from_t1_to_server`
* `generate_from_server_to_t1`
* `generate_from_soc_to_t1`

Maybe we could reduce this part of code to have two methods:
* `generate_downstream_packets`
* `generate_upstream_packets`